capellambse/_namespaces.py

```diff
@@ -181,41 +162,16 @@ def check_plugin(name: str, plugin: Plugin) -> None:
             plugin version range from :class:``capellambse.NAMESPACES``.
     """
     try:
-        my_plugin = NAMESPACES.get_plugin(name)
+        my_plugin = NAMESPACES_PLUGINS[name]
     except KeyError as err:
-        raise UnsupportedPluginError(plugin) from err
+        raise UnsupportedPluginError(f"{plugin}") from err
```

Comment:
This looks weird. Wouldn't `str(plugin)` be cleaner (and possibly even faster)?

```suggestion
        raise UnsupportedPluginError(str(plugin)) from err
```